### PR TITLE
Profiler Timeline View Release

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

- Link to new blog post: https://www.datadoghq.com/blog/continuous-profiler-timeline-view/
- Remove references to "Beta"
- Provide navigation instructions
- Slight re-structure to improve flow

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: static/images/profiler/profiling_viz-timeline3.png
Comment (gturbat) [inline static/images/profiler/profiling_viz-timeline3.png:1]: Let's use 
image
 instead (more accurate to the final UI we have)

File: content/en/profiler/profile_visualizations.md [hunk 1]
```diff
@@ -84,26 +87,28 @@ Four tabs are below the profile header:
...
 - Sort out complex interactions between threads
 - Surface runtime activity that impacted the process
 
+{{< img src="profiler/profiling_viz-timeline3.png" alt="A timeline" >}}
+
+To access the timeline view:
+
+1. Go to [**APM** > **Profiles** > **Explorer**][7].
+1. Select a profile from the **Profile List**.
+1. Set the **Visualize as** option to **Timeline**.
```
Comment (gturbat) [inline content/en/profiler/profile_visualizations.md:110]:
```suggestion
2. Set the **Visualize as** option to **Timeline**.
```

File: content/en/profiler/profile_visualizations.md [hunk 2]
```diff
@@ -84,26 +87,28 @@ Four tabs are below the profile header:
...
 ### Timeline view
 
-The timeline view is the equivalent of the flame graph, with a distribution over time.
-
-{{< img src="profiler/profiling_viz-timeline2.png" alt="A timeline" >}}
-
-It shows time-based patterns and work distribution over:
-- [The period of a single profile](#single-profile)
-- [A trace][6]
+The timeline view is equivalent to the flame graph, with time-based patterns and work distribution over [the period of a single profile](#single-profile) and [a trace][6].
```
Comment (gturbat) [inline content/en/profiler/profile_visualizations.md:96]:
```suggestion
The timeline view is equivalent to the flame graph, with time-based patterns and work distribution over [the period of a single profile][7] and [a trace][6].
```

File: content/en/profiler/profile_visualizations.md [hunk 3]
```diff
@@ -183,3 +188,4 @@ Lanes on the top are runtime activities that may add extra latency to your reque
 [4]: /profiler/profile_types/
 [5]: /dashboards/widgets/profiling_flame_graph
 [6]: /profiler/connect_traces_and_profiles/#span-execution-timeline-view
+[7]: https://app.datadoghq.com/profiling/explorer
```
Comment (gturbat) [inline content/en/profiler/profile_visualizations.md:191]:
```suggestion
[7]: https://app.datadoghq.com/profiling/explorer?viz=thread_timeline
```

**Review by alai97 — APPROVED** [review]
LGTM pending @gturbat's suggestions.

**Pull request merged**